consensus/src/dag.rs

```diff
@@ -429,4 +453,21 @@ impl<PublicKey: VerifyingKey> Dag<PublicKey> {
             .await
             .expect("Failed to receive reply to Remove command from store")
     }
+
+    pub async fn notify_read(
```

Comment:
Could you please add a comment on what the method does and what the caller should expect?

Reply:
done